test/e2e/utils/test_context.go

```diff
@@ -23,7 +23,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo" //nolint:golint
```

Comment:
What linter did you see here?

Reply:
It is because of the `.` as an alias.

> test/e2e/utils/test_context.go:26:2: should not use dot imports (golint)
> 	. "github.com/onsi/ginkgo"
> 	^
> test/e2e/v1/e2e_suite.go:26:2: should not use dot imports (golint)
> 	. "github.com/onsi/ginkgo"
> 	^
> test/e2e/v1/e2e_suite.go:27:2: should not use dot imports (golint)
> 	. "github.com/onsi/gomega"
> 	^
>